src/main/java/cn/nukkit/block/BlockStemMelon.java

```diff
@@ -7,12 +7,21 @@
 import cn.nukkit.level.Level;
 import cn.nukkit.math.BlockFace;
 import cn.nukkit.math.BlockFace.Plane;
+import cn.nukkit.utils.Faceable;
 import cn.nukkit.utils.Utils;
 
+import cn.nukkit.block.custom.properties.BlockProperties;
+import cn.nukkit.block.custom.properties.IntBlockProperty;
+import cn.nukkit.block.properties.BlockPropertiesHelper;
+
 /**
  * Created by Pub4Game on 15.01.2016.
  */
-public class BlockStemMelon extends BlockCrops {
+public class BlockStemMelon extends BlockCrops implements Faceable, BlockPropertiesHelper {
+
+    protected static final IntBlockProperty GROWTH = new IntBlockProperty("growth", false, 13, 0);
+
+    protected static final BlockProperties PROPERTIES = new BlockProperties(GROWTH);
```

Comment:
I think direction should also be added to PROPERTIES instead of being handled separately